nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -28,6 +28,15 @@
      PHP now defaults to PHP 7.4, updated from 7.3.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     Two new options, <literal>authorizedKeysCommand</literal> and
```

Comment:
(nitpick) it would be great if you could linkify those options using `<link linkend="opt-path.to.option">option-name</link>` here.

Reply:
Just built manual and that is **much** better. Thank you for suggestion :heart: